src/gui/SettingsWidget.cpp

```diff
@@ -99,6 +99,12 @@ void SettingsWidget::addSettingsPage(ISettingsPage* page)
 
 void SettingsWidget::loadSettings()
 {
+
+    if (config()->hasAccessError()) {
+        showMessage(
+            tr("Access error for config file ") + config()->getFileName(), MessageWidget::Error);
```

Comment:
You should use a format string. Concatenating strings doesn't work well with translations.